docs/DeveloperGuide.adoc

```diff
@@ -1191,6 +1211,39 @@ Use case resumes at step 5.
 +
 Use case resumes at step 4.
 
+=== Use case: Convert a Lead to a Contact
+
+*MSS*
+
+1. User requests a list of Leads and Contacts
+2. CRM Book shows the requested list
+3. User finds and selects the Lead to convert
+4. CRM Book determines if Lead can be converted
+5. CRM Book converts the Lead to a Contact
```

Comment:
Wrong use of MSS. MSS requires that you must go between user and system. so point 5 cannot be CRM